besu/src/main/resources/org/hyperledger/besu/launcher/launcher.json

```diff
@@ -0,0 +1,102 @@
+[
+    {
+      "prompt-type" :"LIST",
+      "question":"Which ethereum network would you like to use ?",
+      "config-key": "network",
+      "available-options" : "org.hyperledger.besu.cli.config.NetworkName",
+      "additional-flag" : {
+        "yolo_v2": "Xberlin-enabled"
+      }
+    },
+    {
+      "prompt-type" :"LIST",
+      "question":"Which synchronization mode?",
+      "config-key": "sync-mode",
+      "available-options" : "org.hyperledger.besu.ethereum.eth.sync.SyncMode"
+    },
+    {
+      "prompt-type": "CONFIRM",
+      "question": "Do you want to enable pruning?",
+      "config-key": "pruning-enabled",
+      "default-option": "no"
+    },
+    {
+      "prompt-type" :"INPUT",
+      "question":"What is the data directory ?",
+      "config-key" : "data-path"
+    },
+    {
+        "prompt-type" :"CONFIRM",
+        "question":"Do you want to enable the JSON-RPC HTTP service ?",
+        "config-key" : "rpc-http-enabled",
+        "default-option" : "yes",
+        "sub-questions" : [
+               {
+                 "prompt-type" :"CONFIRM",
+                 "question":"Do you want to configure the JSON-RPC options now ?",
+                 "default-option" : "yes",
+                 "sub-questions" : [
+                       {
+                          "prompt-type" :"INPUT",
+                          "question":"What is the JSON RPC HTTP host address ?",
+                          "config-key" : "rpc-http-host"
+                        },
+                        {
+                          "prompt-type" :"INPUT",
+                          "question":"What is the JSON RPC HTTP port ?",
+                          "config-key" : "rpc-http-port"
+                        },
+                        {
+                          "prompt-type" :"CHECKBOX",
+                          "question":"Select the list of APIs to enable on JSON-RPC HTTP service",
+                          "config-key": "rpc-http-apis",
+                          "available-options" : "org.hyperledger.besu.ethereum.api.jsonrpc.RpcApis$ALL_JSON_RPC_APIS"
+                        }
+                   ]
+               }
+            ]
+     } ,
+     {
+        "prompt-type" :"CONFIRM",
+        "question":"Do you want to enable P2P functionality ?",
```

Comment:
Is there a case where we _don't_ want this enabled?

Reply:
no it's true, I just removed it.